bloch_sphere/animate_bloch.py

```diff
@@ -215,7 +215,8 @@ def draw_frame(*args, background='white', idPrefix='d', w=624, h=None,
 def draw_bloch_sphere(d, inner_proj=euclid3d.identity(3), label='', axis=None,
                       rot_proj=None, rot_deg=180,
                       outer_labels=(), inner_labels=(),
-                      extra_opacity=1, inner_opacity=1, background='white'):
+                      extra_opacity=1, inner_opacity=1, background='white',
+                      style=None):
```

Comment:
Later, `None` will break the `.get` check later.  Add a line here `if style is None: style = {}`.

Reply:
No longer needed after `--style` change